### ISSUE TITLE
Accuracy on validation set

### ISSUE THREAD

**Issue by aurooj** [rosinality/mac-network-pytorch#1]
Hi @rosinality,
Is the accuracy you reported for training set?

I got 96.xx% accuracy on train data, and the avg. accuracy on validation set is 85.9%.

**Comment by rosinality** [comment]
Reported value is valid set accuracy. I remember that I haven't experienced severe overfitting.

**Comment by aurooj** [comment]
Thanks for your reply. Oh okay, I will train it again and will try early stopping.

**Comment by geyang** [comment]
@aurooj thanks for this helpful issue! did you have to do anything special to get the validation error up?

**Comment by gitzyong812** [comment]
I run the original code, and also only achieve ~85% acc on val set. Then I remove EMA trick, and the validation acc becomes ~93.5%, still much lower than the reported....
